NextJS: Support @next/font (#20291)
storybookjs/storybook: code/frameworks/nextjs/README.md

This format is not supported by this framework yet. Feel free to [open up an issue](https://github.com/storybookjs/storybook/issues) if this is something you want to see.

### Next.js Font Optimization

[@next/font](https://nextjs.org/docs/basic-features/font-optimization) is partially supported in Storybook. The packages `@next/font/google` and `@next/font/local` are supported.

#### @next/font/google

You don't have to do anything. `@next/font/google` is supported out of the box.

#### @next/font/local

> For local fonts you have to define the [src](https://nextjs.org/docs/api-reference/next/font#src) property.
> The path is relative to the directory where the font loader function is called.
> 
> If the following component defines your localFont like this:

Reviewer: ```suggestion
For local fonts, you have to define the [`src`](https://nextjs.org/docs/api-reference/next/font#src) property.
The path is relative to the directory where the font loader function is called.

For example, if the following component defines a local font like this:
```

```js
// src/components/MyComponent.js
import localFont from '@next/font/local';

const localRubikStorm = localFont({ src: './fonts/RubikStorm-Regular.ttf' });
```

You have to tell Storybook where the `fonts` directory is located:

```js
// .storybook/main.js
module.exports = {
  ...
  "staticDirs": [
    {
      from: '../src/components/fonts',
      to: 'src/components/fonts'
    }
  ],
}

> ```

Reviewer: The simultaneous needs of a path that's relative to the component file _and_ a path that is also correct when building a static Storybook is breaking my brain. I suspect this example needs some explanatory comments.

Author: @kylegach Updated the Readme. Hopefully, it is more straightforward now!
